akka-docs/src/main/paradox/guide/tutorial_5.md

```diff
@@ -76,7 +76,7 @@ not used yet, the built-in scheduler facility. Using the scheduler is simple:
 * We get the scheduler from the `ActorSystem`, which, in turn,
 is accessible from the actor's context: @scala[`context.system.scheduler`]@java[`getContext().getSystem().scheduler()`]. This needs an @scala[implicit] `ExecutionContext` which
 is the thread-pool that will execute the timer task itself. In our case, we use the same dispatcher
-as the actor by @scala[importing `import context.dispatcher`] @java[passing in `getContext().dispatcher()`].
+as the actor by @scala[using `private implicit val ec: ExecutionContext = context.dispatcher`] @java[passing in `getContext().dispatcher()`].
```

Comment:
This one is not problematic, as you wouldn't schedule something new after the actor has stopped and the scheduling is what is picking up the implicit ec.